eureka-client/src/main/java/com/netflix/discovery/shared/transport/decorator/RedirectingEurekaHttpClient.java

```diff
@@ -80,15 +80,15 @@ public void shutdown() {
                 TransportUtils.shutdown(delegateRef.getAndSet(currentEurekaClientRef.get()));
                 return response;
             } catch (Exception e) {
-                logger.error("Request execution error. endpoint={}", serviceEndpoint, e);
+                logger.info("Request execution error. endpoint={}", serviceEndpoint);
```

Comment:
Same, exception details and we can elaborate on the details.

Reply:
I realize in this case, we *are* losing the exception message, so I'll ensure I add it here.